client/app/bundles/course/assessment/submission/pages/LogsIndex/LogsHead.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC } from 'react';
+import { Link as ReactLink } from 'react-router-dom';
+import { TableBody, TableCell, TableRow } from '@mui/material';
+import { LogsMainInfo } from 'types/course/assessment/submission/logs';
+
+import Link from 'lib/components/core/Link';
+import { TableContainer } from 'lib/components/core/table';
+import useTranslation from 'lib/hooks/useTranslation';
+
+import translations from './translations';
+
+interface Props {
+  with: LogsMainInfo;
+}
+
+const LogsHead: FC<Props> = (props) => {
+  const { t } = useTranslation();
+  const { with: info } = props;
+
+  return (
+    <TableContainer dense variant="outlined">
+      <TableBody>
+        <TableRow>
+          <TableCell>{t(translations.assessmentTitle)}</TableCell>
+          <TableCell>
+            <ReactLink to={info.assessmentUrl}>
+              {info.assessmentTitle}
+            </ReactLink>
+          </TableCell>
+        </TableRow>
+
+        <TableRow>
+          <TableCell>{t(translations.studentName)}</TableCell>
+          <TableCell>
+            <Link href={info.studentUrl}>{info.studentName}</Link>
+          </TableCell>
+        </TableRow>
+
+        <TableRow>
+          <TableCell>{t(translations.submissionStatus)}</TableCell>
+          <TableCell>
+            <ReactLink to={info.editUrl}>{info.submissionStatus}</ReactLink>
```

Comment:
Does this happen non non-password assessment?

Reply:
okay have changed it into chips.. as for that alert, the opening of that depends on the value (boolean) of the props `passwordProtected` inside the `SubmissionEditForm`.  Which means, this only shows for the password-protected assessment

Reply:
Yes, but the problem that I reported is not that this alert opens on all kinds of submissions. I was reporting that it appears when I open _any password-protected submissions_. It shouldn't happen because I'm not attempting.

It doesn't happen on production.

Reply:
yes I am able to replicate, and I already know how to fix it.. Just to confirm, so the expected behavior is that the alert is supposed to be present when user is attempting and also when it's password-protected, right?

If so, then I will fix it accordingly and push it directly

Reply:
Ah ok. The popup was only shown only when "submission=new" params is included but i removed that since we dont have it anymore. @bivanalhar could you check if the user who accesses the page is the author of the submission then show it?

Reply:
@ekowidianto before I modified the code inside the `SubmissionEditForm`, if the author of the submission accesses the edit page, he/she will see that alert. However, after the modification, the alert won't be shown if the user's stance is not Attempting

<img width="1440" alt="Screenshot 2023-04-12 at 4 50 01 PM" src="https://user-images.githubusercontent.com/16359075/231405119-2bafbb7a-60b2-401a-b773-b9cd593773ec.png">

Reply:
@bivanalhar you are right